### PR TITLE
Move to trending playlists and show favorites

### PR DESCRIPTION
**Pull request by raymondjacobson**

### CODE REVIEW

**Comment by raymondjacobson** [comment]
large diff is b/c of M1 npm install

File: src/types.ts [hunk 1]
```diff
@@ -258,5 +260,7 @@ export type Album = {
   handle: string
   artwork: string
   plays: number
+  favorites: number,
+  trackCount: number
```
Comment (sddioulde) [inline src/types.ts:264]: Nit. Prop names could be consistent. If we're using the Count suffix maybe update other vars too

**Review by sddioulde — APPROVED** [review]

File: src/store/cache/music/hooks.ts [hunk 2]
```diff
@@ -88,6 +90,7 @@ export function fetchTopAlbums(
         handle: d.user.handle,
         artwork: d.artwork?.['480x480'] ?? imageBlank,
         plays: d.total_play_count,
+        favorites: d.favorite_count,
```
Comment (sddioulde) [inline src/store/cache/music/hooks.ts:93]: Do we care about trackCount here too?

Comment (raymondjacobson) [reply src/store/cache/music/hooks.ts]: we actually don't. i removed all of these vars. we decided better to just not show anything

**Pull request merged**